src/viper/trans.ml

```diff
@@ -5,6 +5,23 @@ open Syntax
 module T = Mo_types.Type
 module M = Mo_def.Syntax
 
+module Stamps = Env.Make(String)
+
+let stamps : int Stamps.t ref = ref Stamps.empty
+
+let (!!) it = { it; at = Source.no_region; note = NoInfo}
```

Comment:
thank you :-)

Reply:
```suggestion
let (!!) it = { it; at = Source.no_region; note = NoInfo }
```

;-)

Reply:
Btw, what's your intuition for calling this symbol `!!`?

Reply:
None.